lib/configproxy.js

```diff
@@ -145,7 +145,7 @@ class ConfigurableProxy extends EventEmitter {
     } else {
       this.includePrefix = true;
     }
-    this.customHeaders = this.options.customHeaders;
+    this.headers = this.options.headers;
```

Comment:
the `customHeaders` name is still used on L232. Can you remove it?

Reply:
Woops, nice catch. Note to self: always search for something I replace to spot if I missed something.

Reply:
Hmmm... so this is a part of the constructor of ConfigurableProxy

```node
    // proxy websockets
    this.proxyServer.on("upgrade", bound(this, this.handleProxyWs));

    this.proxy.on("proxyRes", function(proxyRes, req, res) {
      that.statsd.increment("requests." + proxyRes.statusCode, 1);
      if (that.customHeaders) {
        that.customHeaders.forEach(header => res.setHeader(header.key, header.value));
      }
    });
```

If I replace this with `that.header` I get errors I get socket hang up errors and I'm not sure why. Debugging.

Reply:
I'm not sure what this code is meant to do or what the "proxyRes" event is, is that a response event to a request made? Then it doesn't make sense to modify the response.

Hmm...

Reply:
Also, httpProxy should be configured already.

I'll delete the section, perhaps that is what you meant with "can you remove it" rather than "can I replace the use of customHeaders with headers".

Reply:
I deleted that section, which previously have done nothing, and seem to contain logic of no use.